1.0/tools/optimize-for-production.md

```diff
@@ -0,0 +1,268 @@
+---
+layout: default
+type: guide
+shortname: Docs
+title: Optimize for production
+subtitle: Developer guide
+---
+
+{% include toc.html %}
+
+Reducing network requests is important for a performant app experience. In the {{site.project_title}} world, [Vulcanize](https://github.com/Polymer/vulcanize) is the name given to a build tool that lets you **concatenate** a set of elements and their HTML imported dependencies into a single file. Vulcanize recursively pulls in all your imports, flattens their dependencies and spits out something that can **reduce the number of network requests** your app makes.
+
+**Note:** for more great info on performance considerations worth keeping in mind when using HTML Imports, see [HTML Imports - #include for the web](http://www.html5rocks.com/en/tutorials/webcomponents/imports/#performance)
+{: .alert .alert-info }
+
+## Installation
+
+Install Vulcanize and its dependencies using [NPM](http://npmjs.org):
+
+    npm install -g vulcanize
+
+We recommend installing Vulcanize globally so you can run it from anywhere on the command-line. To install locally, simply omit the `-g` flag.
+
+## Usage
+
+Vulcanize can be used standalone from the command line, or as part of a gulp/grunt build chain.
+
+### Use vulcanize from the command line
+
+Assuming an input HTML file, elements.html, using a number of HTML imports, we can run it through Vulcanize by passing it as an argument as follows:
+
+    vulcanize elements.html
+
+The output will be printed to stdout in the terminal.
+
+To write the output to a file, use the `--output` or `-o` flag:
+
+    vulcanize -o elements.vulcanized.html elements.html
+
+No additional configuration is necessary. elements.vulcanized.html now contains a version of elements.html with all imports inlined and dependencies flattened. Paths to any URLs are automatically adjusted for the new output location, with the exception of those set in JavaScript.
```

Comment:
plz format your filenames throughout the doc as inline code. e.g. `elements.vulcanized.html`